packages/esm-ohri-pmtct-app/src/views/mch-summary/tabs/current-pregnancy.component.tsx

```diff
@@ -149,14 +172,34 @@ const CurrentPregnancy: React.FC<PatientChartProps> = ({ patientUuid }) => {
     return items;
   }, [relatives, relativeToIdentifierMap]);
 
+  const childrenDetails: pregnancyOutcomeProps[] = useMemo(() => {
+    let items = [];
+    pregnancyOutcomes.forEach((child) => {
+      let infantStatusObs = child.groupMembers.find((member) => member.concept.uuid === infantStatusAtBirthConcept);
+      let childObject: pregnancyOutcomeProps = {
+        id: child.groupMembers.find((member) => member.concept.uuid === infantPTrackerIdConcept).value,
```

Comment:
This line looks like the data being displayed is actually the `pTracker` and not the id (uuid). My view is that u change this to pTrackerId for consistency